scripts/environment/bootstrap-ubuntu-20.04.sh

```diff
@@ -3,7 +3,11 @@ set -e -o verbose
 
 export DEBIAN_FRONTEND=noninteractive
 
+apt-get install -y grub-efi
```

Comment:
Not a big deal in this particular case, but I generally prefer to use the longer form of flags, that is `--yes`, in scripts to make them easier to read without pulling up the man page.